pkg/image/admission/imagepolicy/api/validation/validation.go

```diff
@@ -27,6 +27,12 @@ func Validate(config *api.ImagePolicyConfig) field.ErrorList {
 		}
 	}
 
+	for i, rule := range config.ResolutionRules {
+		if len(rule.TargetResource.Resource) == 0 {
```

Comment:
I was considering getting a list of resources registered in schema, but I'm guessing this is not the right approach :/